lua/telescope/make_entry.lua

```diff
@@ -612,31 +612,41 @@ function make_entry.gen_from_packages(opts)
   end
 end
 
-function make_entry.gen_from_apropos()
+function make_entry.gen_from_apropos(opts)
+  local sections = {}
+  for _, section in ipairs(vim.split(opts.sections, ',', true)) do
+    sections[section] = true
+  end
+
   local displayer = entry_display.create {
-    separator = "",
+    separator = ' ',
     items = {
-      { width = 30 },
-      { remaining = true },
+      {},
+      {},
     },
   }
 
   local make_display = function(entry)
+    -- Stopgap measure to set `width` for this field. See #414
+    local formatted = ('%-30s'):format(entry.keyword)
     return displayer {
-      entry.value,
+      {formatted, 'TelescopeResultsFunction'},
       entry.description
     }
   end
 
   return function(line)
-    local cmd, _, desc = line:match("^(.*)%s+%((.*)%)%s+%-%s(.*)$")
-
-    return {
+    local keyword, desc = line:match'^(.+)%s+%-%s+(.*)$'
+    local section = keyword and keyword:match'%(([^)])%)' or nil
```

Comment:
rather then `[^)]` why not `%d`

Reply:
Sections in man pages may have non-digit characters such as `1m`, `x`, etc. ([man page - Wikipedia][])

[man page - Wikipedia]: https://en.wikipedia.org/wiki/Man_page

On my Mac, many pages have such sections.

```
apropos ' ' | egrep -v '\(\d+\)' | head

TIFFClose(3tiff)         - close a previously opened TIFF file
TIFFDataWidth(3tiff)     - Get the size of TIFF data types
TIFFError(3tiff), TIFFSetErrorHandler(3tiff) - library error handling interface
TIFFFieldDataType(3tiff) - Get TIFF data type from field information
TIFFFieldName(3tiff)     - Get TIFF field name from field information
TIFFFieldPassCount(3tiff) - Get whether to pass a count to TIFFGet/SetField
TIFFFieldReadCount(3tiff) - Get number of values to be read from field
TIFFFieldTag(3tiff)      - Get TIFF field tag value from field information
TIFFFieldWriteCount(3tiff) - Get number of values to be written to field
TIFFFlush(3tiff), TIFFFlushData(3tiff) - flush pending writes to an open TIFF file
```

Reply:
You learn something every day :)